docker/gpu.Dockerfile

```diff
@@ -9,10 +9,16 @@ RUN rm -r /opt/conda/lib/cmake/Boost-1.72.0
 RUN apt-get update
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get -y install libsdl2-gfx-dev libsdl2-ttf-dev libsdl2-image-dev xorg
-ADD ./docker/envs/football/egl /tmp/football
-RUN cd /tmp/football && \
+RUN cd /tmp && \
+    git clone --single-branch --branch v2.8 https://github.com/google-research/football.git && \
+    cd football && \
+    sed -i 's/copy2/move/g' gfootball/env/observation_processor.py && \
```

Comment:
BTW I ended up testing without these lines and you're right -- it seems like everything works fine without them now.